app/models/chargeback_rate_detail.rb

```diff
@@ -155,16 +156,14 @@ def hourly_cost(value, consumption)
   end
 
   def hourly(rate, consumption)
-    hourly_rate = case per_time
+    case per_time
                   when "hourly"  then rate
                   when "daily"   then rate / 24
                   when "weekly"  then rate / 24 / 7
                   when "monthly" then rate / consumption.hours_in_month
                   when "yearly"  then rate / 24 / 365
                   else raise "rate time unit of '#{per_time}' not supported"
                   end
```

Comment:
not sure if it is in this PR.
these need to be outdented

Reply:
Yeah, I can fix that manually... Because I'm not running all the cops, sometimes "partial" fixes occur